specification/maps/resource-manager/Microsoft.Maps/preview/2023-12-01-preview/examples/PrivateEndpointConnections_Delete.json

```diff
@@ -0,0 +1,23 @@
+{
+  "parameters": {
+    "api-version": "2023-12-01-preview",
+    "subscriptionId": "21a9967a-e8a9-4656-a70b-96ff1c4d05a0",
+    "resourceGroupName": "myResourceGroup",
+    "accountName": "myMapsAccount",
+    "privateEndpointConnectionName": "{privateEndpointConnectionName}",
+    "monitor": "true"
+  },
+  "responses": {
+    "200": {},
+    "202": {
+      "headers": {
+        "Azure-AsyncOperation": "https://management.azure.com/subscriptions/21a9967a-e8a9-4656-a70b-96ff1c4d05a0/resourceGroups/myResourceGroup/providers/Microsoft.Maps/accounts/myMapsAccount/operations/2d7f573a-7acd-4a3c-9425-fed7d029c69d?api-version=2023-12-01-preview",
+        "Location": "https://management.azure.com/subscriptions/21a9967a-e8a9-4656-a70b-96ff1c4d05a0/providers/Microsoft.Maps/locations/{location}/operationResults/2d7f573a-7acd-4a3c-9425-fed7d029c69d?api-version=2023-12-01-preview",
```

Comment:
We should double check that we're picking the right schema for async and location headers.